### ISSUE TITLE
Keepass2Android Offline app/process/service runs in the background even after exiting the application.

### ISSUE THREAD

**Issue by LoneRan-ya** [PhilippC/keepass2android#1961]
Hi,
I'm using Keepass2Android Offline version 1.08d on an Android 11 phone.
I had expected that once I'm done using this application and then exiting it,  the application would no longer be running in the background.
But what I'm seeing on my phone is after I exit the application the app process is still running.
In my K2A settings I've also enabled the "Close-Button" to kill the app process (I am one of those 'paranoid' users).
So, how am I checking that the app process is still running? - In Developer options I look at the "Running services" option and I see "Keepass2Android Offline" listed there with the message '1 process and 1 service'.
So I've also checked the same "Running services" setting right after I restart the phone or even if I power it Power it off and On again, and without even opening the Keepass2Android app it still shows that same process is always running.
I have also tried to manually "force stop" the application via -> Settings -> Apps & Notifications -> Keepass2Android Offline -> Force stop,   but without success. The app just won't die.
It behaves like a system application.
Can you please explain why this might be happening. I don't want this app/process running in the background once I'm done using the application. 

BTW I do like this app. Have been using it for many years on a much older phone, but now it's the first time I'm using the latest version of the app on a newer phone.
cheers,
Ricardo

**Comment by PhilippC** [comment]
are you using the KP2A keyboard?

**Comment by LoneRan-ya** [comment]
Well Philipp, you gave me the hint on what area to look under and I've found the reason. The cause was related to the KP2A keyboard as you suspected.
I had thought that in order to NOT be using the KP2A keyboard it would be sufficient to change it in the KP2A settings by unchecking the option  under -> app -> security -> 'Use built-in keyboard inside Keepass2Andrroid' which I had done.
 But then I also checked in the Android phone settings itself under :  System -> Languages and Input -> On-screen keyboard and saw that the default keyboard had been set to "Keepass2Android Offline" (with the other keyboard  'Gboard' not enabled or not available). 
So I re-enabled  'Gboard' as an input method (in addition to KP2A), then went back to the KP2A settings and enabled 'Use built-in keyboard inside Keepass2Andrroid' again.
So now whenever I start to input text within the KP2A app, I get asked on which keyboard to use. So I choose the KP2A keyboard, make my changes and then exit the application and then go and check the "Running services" again and I'm glad to see that KP2A is no longer running.
Not sure how KP2A became the system-wide default but I guess it was probably my doing at some point.
Thanks for the reply.
Ricardo
